sdks/python/apache_beam/runners/worker/sideinputs.py

```diff
@@ -51,9 +52,12 @@ class PrefetchingSourceSetIterable(object):
   """Value iterator that reads concurrently from a set of sources."""
 
   def __init__(self, sources,
-               max_reader_threads=MAX_SOURCE_READER_THREADS):
+               max_reader_threads=MAX_SOURCE_READER_THREADS,
+               read_counter=None):
     self.sources = sources
     self.num_reader_threads = min(max_reader_threads, len(self.sources))
+    self.read_counter = read_counter or opcounters.TransformIoCounter()
+    # self.read_counter = opcounters.TransformIoCounter()
```

Comment:
<!--new_thread; commit:6e2c396b4cdcce20867c583cba65c8664ff79996; resolved:0-->
Why is this commented out ?

Reply:
This is to allow to test a no-op counter vs the implementation. It will be removed before merging.